language/stdlib/modules/0L/LibraAccount.move

```diff
@@ -914,8 +915,13 @@ module LibraAccount {
     ): WithdrawCapability acquires LibraAccount {
         //////// 0L //////// Transfers disabled by default
         //////// 0L //////// Transfers of 10 GAS 
-        //////// 0L //////// enabled when validator count is 100. 
+        //////// 0L //////// enabled when epoch is 1000. 
         let sender_addr = Signer::address_of(sender);
+
+        // Community wallets have own transfer mechanism.
+        let community_wallets = Wallet::get_comm_list();
+        assert(!Vector::contains(&community_wallets, &sender_addr), Errors::limit_exceeded(EWITHDRAWAL_EXCEEDS_LIMITS));
```

Comment:
Yes, I agree it's inefficient. But I don't think it gets used often. I opted to not have a tag on the account because we do need a list of community wallets, for times we need to iterate. Then having an additional struct on an account to tag it as a community wallet seemed excessive. But we can do that.